### ISSUE TITLE
histoire story collection broken in Nuxt 3.8.0

### ISSUE THREAD

**Issue by piscis** [histoire-dev/histoire#616]
### Describe the bug

The story collection seems to be broken in nuxt 3.8.0 same code works with nuxt 3.7.4

```bash
[nuxt] Calling `useRoute` within middleware may lead to misleading results. Instead, use the (to, from) arguments passed to the middleware to access the new and old routes.
<Suspense> is an experimental feature and its API will likely change.
Error while collecting story /home/projects/nuxt-starter-tm4z58/DemoComponent.story.vue:
FetchError: [GET] "/builds/meta/test.json": 404 Cannot find any path matching /builds/meta/test.json.
    at async $fetch2 (/home/projects/nuxt-starter-tm4z58/node_modules/ofetch/dist/shared/ofetch.00501375.mjs:278:15)
```

### Reproduction

https://stackblitz.com/edit/nuxt-starter-tm4z58?file=package.json

### System Info

```shell
System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 0 Bytes / 0 Bytes
    Shell: 1.0 - /bin/jsh
  Binaries:
    Node: 18.18.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 9.4.2 - /usr/local/bin/npm
    pnpm: 8.9.2 - /usr/local/bin/pnpm
  npmPackages:
    @histoire/plugin-nuxt: ^0.17.3 => 0.17.3 
    @histoire/plugin-vue: ^0.17.1 => 0.17.1 
    @vitejs/plugin-vue: ^4.3.4 => 4.4.0 
    histoire: ^0.17.2 => 0.17.2
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/histoire-dev/histoire/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/histoire-dev/histoire/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://histoire.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/histoire-dev/histoire/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/histoire-dev/histoire/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by MGolob** [comment]
Same here. Fresh project, installed histoire and it failes with that error message.

**Comment by medfreeman** [comment]
it was fixed in [`@histoire/plugin-nuxt` v0.17.4](https://github.com/histoire-dev/histoire/releases/tag/v0.17.4)

**Comment by hacknug** [comment]
Not sure that was the same issue. Maybe OP can let us know if their issue was solved?
